Riot/Modules/Room/RoomViewController.m

```diff
@@ -567,35 +567,6 @@ - (void)viewDidAppear:(BOOL)animated
         }
     }];
     [self refreshMissedDiscussionsCount:YES];
-    
-    // Warn about the beta state of e2e encryption when entering the first time in an encrypted room
-    MXKAccount *account = [[MXKAccountManager sharedManager] accountForUserId:self.roomDataSource.mxSession.myUser.userId];
-    if (account && !account.isWarnedAboutEncryption && self.roomDataSource.room.summary.isEncrypted)
```

Comment:
Maybe we could now add a deprecated annotation on `MXKAccount.isWarnedAboutEncryption` property.

Reply:
Even remove it. The branch is ready. I just want to check with Giom before.